app/navigation/NomineesNavigator.scala

```diff
@@ -65,7 +65,11 @@ class NomineesNavigator @Inject()(implicit frontendAppConfig: FrontendAppConfig)
     }
 
     case IndividualNomineesPassportPage => userAnswers: UserAnswers => userAnswers.get(IndividualNomineesPassportPage) match {
-      case Some(_) => controllers.addressLookup.routes.NomineeIndividualAddressLookupController.initializeJourney(NormalMode)
+      case Some(_) => if(frontendAppConfig.isExternalTest){
+        routes.DeadEndController.onPageLoad() // TODo Summary Pages
```

Comment:
we seem to have missed this one and line 78, but shouldn't this be redirecting to the 'IsIndividualNomineePaymentsPage', not to the summary? that could be implemented now.